src/lib/Containers/__tests__/Conversation-tests.tsx

```diff
@@ -26,7 +26,7 @@ const props = {
             raw_text: "Adoro! Por favor envie-me mais informações",
             from_email_address: "anita@garibaldi.br",
             attachments: [],
-            created_at: "2017-06-26T14:14:35.538Z",
+            last_message_at: moment().subtract(2, "days").toISOString(),
```

Comment:
Ah shoot I think I added that the other day (and undid the correct relative timestamp). Thank ye